datastore/gcloud/aio/datastore/datastore.py

```diff
@@ -346,3 +346,51 @@ def make_mutation(operation: Operation, key: Key,
         transaction = await self.beginTransaction(session=session)
         mutation = self.make_mutation(operation, key, properties=properties)
         await self.commit([mutation], transaction=transaction, session=session)
+
+    async def export(self, output_bucket_name: str,
```

Comment:
Let's call this param `output_bucket_prefix`, since it supports arbitrary prefix-based folder nesting.

Reply:
updated